airtime_mvc/application/controllers/LibraryController.php

```diff
@@ -280,6 +282,7 @@ public function deleteAction()
     public function duplicateAction(){
         $params = $this->getRequest()->getParams();
         $id = $params['id'];
+        Logging::info($params);
```

Comment:
Should this be `Logging::debug`? If `Logging::debug` is even a thing, I didn't check...

We need to clean up the logs anyhow so I won't consider this a blocker for this PR 😉